response/types.go

```diff
@@ -649,13 +649,291 @@ type SearchUserResponse struct {
+				CreatedAt    int    `json:"created_at"`
+				CreatedAtUtc int    `json:"created_at_utc"`
+				ContentType  string `json:"content_type"`
+				Status       string `json:"status"`
+				BitFlags     int    `json:"bit_flags"`
+				User         struct {
+					Pk               int    `json:"pk"`
+					Username         string `json:"username"`
+					FullName         string `json:"full_name"`
+					IsPrivate        bool   `json:"is_private"`
+					ProfilePicURL    string `json:"profile_pic_url"`
+					ProfilePicID     string `json:"profile_pic_id"`
+					FriendshipStatus struct {
+						Following       bool `json:"following"`
+						OutgoingRequest bool `json:"outgoing_request"`
+						IsBestie        bool `json:"is_bestie"`
+					} `json:"friendship_status"`
+					IsVerified                 bool `json:"is_verified"`
+					HasAnonymousProfilePicture bool `json:"has_anonymous_profile_picture"`
+					IsUnpublished              bool `json:"is_unpublished"`
+					IsFavorite                 bool `json:"is_favorite"`
+					LatestReelMedia            int  `json:"latest_reel_media"`
+				} `json:"user"`
+				DidReportAsSpam bool  `json:"did_report_as_spam"`
+				MediaID         int64 `json:"media_id"`
+				HasTranslation  bool  `json:"has_translation"`
+			} `json:"caption"`
+			CaptionIsEdited              bool          `json:"caption_is_edited"`
+			LikeCount                    int           `json:"like_count"`
+			HasLiked                     bool          `json:"has_liked"`
+			TopLikers                    []interface{} `json:"top_likers"`
+			CommentLikesEnabled          bool          `json:"comment_likes_enabled"`
+			CommentThreadingEnabled      bool          `json:"comment_threading_enabled"`
+			HasMoreComments              bool          `json:"has_more_comments"`
+			MaxNumVisiblePreviewComments int           `json:"max_num_visible_preview_comments"`
+			PreviewComments              []interface{} `json:"preview_comments"`
+			CommentCount                 int           `json:"comment_count"`
+			PhotoOfYou                   bool          `json:"photo_of_you"`
+			CanViewerSave                bool          `json:"can_viewer_save"`
+			OrganicTrackingToken         string        `json:"organic_tracking_token"`
+			Algorithm                    string        `json:"algorithm"`
+			ExploreContext               string        `json:"explore_context"`
+			ExploreSourceToken           string        `json:"explore_source_token"`
+			Explore                      struct {
+				Explanation string `json:"explanation"`
+				ActorID     int    `json:"actor_id"`
+				SourceToken string `json:"source_token"`
+			} `json:"explore"`
+			ImpressionToken string `json:"impression_token"`
+		} `json:"media,omitempty"`
+	} `json:"items"`
 	NumResults          int    `json:"num_results"`
+	RankToken           string `json:"rank_token"`
 	AutoLoadMoreEnabled bool   `json:"auto_load_more_enabled"`
-	Items               []Item `json:"items"`
```

Comment:
I don't want to use inline structs in another structs.
So please update `Item` , Can you ?

Reply:
Hello @ahmdrz . 
Sorry for late answer.
The Item struct in the ExploreResponse is incompatible with old the old used ItemStruct
May be better to make a separate structure ExploreItem  ?

Reply:
Are you sure there is one or more difference between Item and your ExploreItem struct ?
If you sure , Go for it and make ExploreItem.

Reply:
Yes Item struct in the ExploreResponce has changed a lot. I dump a raw json document and save it in the my DropBox. 
https://www.dropbox.com/s/8wn9s68tenvku76/data.json?dl=0
You can download this file to see the changes
[data.json.txt](https://github.com/ahmdrz/goinsta/files/1657554/data.json.txt)